uf3/forcefield/calculator.py

```diff
@@ -387,14 +417,8 @@ def relax_fmax(self,
         return geom
 
     def calculation_required(self, atoms: ase.Atoms, quantities: List) -> bool:
-        """Check if a calculation is required."""
-        if any([q in quantities for q in ['magmom', 'stress', 'charges']]):
-            return True
-        if 'energy' in quantities and 'energy' not in atoms.info:
-            return True
-        if 'force' in quantities and 'fx' not in atoms.arrays:
-            return True
-        return False
+        """Check if a calculation is required."""        
+        return True
```

Comment:
This works as a temporary fix, but I would prefer not forcing the re-compute every time for a permanent solution. You could replace the last line `return False` with a call to the super method that checks if atoms changes. This way, the computations should only execute when they are required.